PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/container/webapponlinux/ui/SettingPanel.java

```diff
@@ -52,6 +53,7 @@
 import com.microsoft.tooling.msservices.serviceexplorer.azure.container.WebAppOnLinuxDeployPresenter;
 import com.microsoft.tooling.msservices.serviceexplorer.azure.container.WebAppOnLinuxDeployView;
 
+import io.netty.util.internal.StringUtil;
```

Comment:
Use `StringUtil.isBlank/isEmpty` see: http://commons.apache.org/lang/ , which is more often used. Avoid to use `*.internal.*` if possible.

Reply:
Replace it with `org.jsoup.helper.StringUtil`